challenge.js

```diff
@@ -42,6 +44,7 @@
  */
 
 // ANSWER
+#data-table-1 :A E {font-weight: bolder;}
```

Comment:
Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.